pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -443,15 +443,15 @@ func (k *Bootstrapper) needsReset(conf string, hostname string, port int, client
 	}
 
 	if err := kverify.ExpectAppsRunning(client, kverify.AppsRunningList); err != nil {
-		glog.Infof("needs reset: %v", err)
+		glog.Infof("needs reset because expected components to be running : %v", err)
```

Comment:
This seems too difficult to read. Is the error retutrned by ExpectAppsRunning not meaningful already?

If not, just use: `needs reset: expected apps: %v`.